Project-II-Calculator/calculator/src/calculation/calculate.js

```diff
@@ -0,0 +1,115 @@
+// ideally for a good calculator the operate function should use a library like
+// decimal.js or big.js since using floating point for calculator functions
+// is inaccurate at tiny values and for rounding.
+
+function operate(one, two, operation) {
```

Comment:
good job thinking beyond integers
I like the commenting here that clearly states what each chunk is doing. Another programmer could step in and see what problems you were trying to solve and why you approached them as you did.